frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.tsx

```diff
@@ -534,6 +536,7 @@ export const TableInteractiveInner = forwardRef(function TableInteractiveInner(
     settings,
     tableTheme,
     isDashboard,
+    clicked,
```

Comment:
I think it's not good to have clicked here as that would cause entire `columnsOptions` to recompute everytime a user clicks on a cell...

@alxnddr do you have any suggestions on what we can use here instead to know that the header has been clicked?